lib/auth/usertoken.go

```diff
@@ -247,6 +247,7 @@ func (s *Server) RotateUserTokenSecrets(ctx context.Context, tokenID string) (ty
 	}
 	secrets.SetOTPKey(key.Secret())
 	secrets.SetQRCode(otpQRBuf.Bytes())
+	secrets.SetExpiry(token.Expiry())
```

Comment:
Should we get test coverage for the missing expiry, so we can avoid regressions in this case?

Reply:
added test for user token secret creation